apps/web/src/components/settings/ClearPendingTxs/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { usePendingTxIds } from '@/hooks/usePendingTxs'
+import { SETTINGS_EVENTS, trackEvent } from '@/services/analytics'
+import { useAppDispatch } from '@/store'
+import { clearPendingTx } from '@/store/pendingTxsSlice'
+import { Stack, Typography, Box, Button, Alert } from '@mui/material'
+import { useCallback } from 'react'
+
+export const ClearPendingTxs = () => {
+  const pendingTxIds = usePendingTxIds()
+  const pendingTxCount = pendingTxIds.length
+  const dispatch = useAppDispatch()
+
+  const clearPendingTxs = useCallback(() => {
+    pendingTxIds.forEach((txId) => {
+      dispatch(clearPendingTx({ txId }))
+    })
+    trackEvent(SETTINGS_EVENTS.DATA.CLEAR_PENDING_TXS)
```

Comment:
What do you think about including the number? Afaiu, we are also aware of the chain so we can isolate troublesome chains easier.